tensorflow_hub/resolver.py

```diff
@@ -62,7 +64,7 @@ class ModelLoadFormat(enum.Enum):
 _TFHUB_CACHE_DIR = "TFHUB_CACHE_DIR"
 _TFHUB_DOWNLOAD_PROGRESS = "TFHUB_DOWNLOAD_PROGRESS"
 _TFHUB_MODEL_LOAD_FORMAT = "TFHUB_MODEL_LOAD_FORMAT"
-
+_TFHUB_DISABLE_CERT_VALIDATION = "TFHUB_DISABLE__CERT_VALIDATION"
```

Comment:
Can we add a comment why this is necessary and that we only disable cert validation if the variable's value is set to "true"? For that, it might make sense to add `_TFHUB_DISABLE_CERT_VALIDATION_VALUE = "true"` below this line.